pq-crypto/bike/sha384.h

```diff
@@ -0,0 +1,110 @@
+/***************************************************************************
+* Additional implementation of "BIKE: Bit Flipping Key Encapsulation". 
+* Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Written by Nir Drucker and Shay Gueron
+* AWS Cryptographic Algorithms Group
+* (ndrucker@amazon.com, gueron@amazon.com)
+*
+* The license is detailed in the file LICENSE.txt, and applies to this file.
+* ***************************************************************************/
+
+#pragma once
+
+#include <stdlib.h>
+#include "types.h"
+
+
+#ifdef AVX512
+    #define NUM_OF_BLOCKS_IN_MB 8ULL
+#else
+    #define NUM_OF_BLOCKS_IN_MB 4ULL
+#endif
+
+#define SLICE_REM           111ULL
+#define MAX_MB_SLICES       8ULL
+#define HASH_BLOCK_SIZE     128ULL
+
+typedef sha384_hash_t sha_hash_t;
+typedef uint64_t block_t;
+
+#define INIT_HASH {0xcbbb9d5dc1059ed8, \
+                   0x629a292a367cd507, \
+                   0x9159015a3070dd17, \
+                   0x152fecd8f70e5939, \
+                   0x67332667ffc00b31, \
+                   0x8eb44a8768581511, \
+                   0xdb0c2e0d64f98fa7, \
+                   0x47b5481dbefa4fa4}
+
+#define K {0x428a2f98d728ae22, 0x7137449123ef65cd, 0xb5c0fbcfec4d3b2f, 0xe9b5dba58189dbbc, \

```

Comment:
Don't think we should be reimplementing SHA hashing for PQ Crypto. I don't think this is used anywhere so I think we can remove it. https://tools.ietf.org/html/rfc6234#section-5.2